examples/user_guide/Customizing_Plots.ipynb

```diff
@@ -154,6 +154,84 @@
     "(hv.Curve([1, 2, 3], label='A') * hv.Curve([3, 2, 1], label='B')).opts(fontscale=2, width=500, height=400, title='Title')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Applying customizations to legends\n",
+    "\n",
+    "When overlaying plots with different labels, a legend automatically appears to differentiate elements in the overlay. This legend can be customized in several ways:\n",
+    "\n",
+    "- by **position**\n",
+    "    - by adjusting the legend location within the figure using the `legend_position` option (e.g. `legend_position='bottom_right'`)\n",
+    "    - by adjusting the legend location *outside* of the figure using the `legend_position` and `legend_offset` parameters (which then positions the legend in *screen* space) (e.g. `legend_position='right', legend_offset=(0, 200)`)\n",
+    "- by **style**\n",
+    "    - by muting elements with `legend_muted=True`\n",
+    "    - by putting the legend elements in a column layout with `legend_cols=True`\n",
```

Comment:
Maybe note which of these options are specific to bokeh. I believe legend_position is supported by matplotlib as well and `legend_cols` even takes an integer in matplotlib. The other options are not available in bokeh.

Reply:
I've tried all examples with matplotlib and I came to the conclusion legend_muted and legend_cols only works with bokeh.
Also legend_offset is bokeh_only too.
I've added some texts to reflect this.

Reply:
IIRC `legend_cols` works as an integer in matplotlib, i.e. it actually specifies the number of columns, something I hope to implement for bokeh eventually.